### DISCUSSION TITLE
Hide login form - locked out

### DISCUSSION THREAD

**Discussion by amaruzzo** [kimai/kimai#4873] [General]
Hi there,

I've de-activated this option (login form) to use only SAML authentication (Azure SAML)
Before that I had given the System-Admin permission to one SAML user.
Obviously I had tried to login, before de-activated the login form option, with the SAML user and I got the right system permissions.

But just I de-activated the form login the SAML user lost the system permissions.
How can I re-activated the option **by command line**? (because I can't login with my local admin user without the login form).

Thanks a lot.

**Comment by amaruzzo** [comment]
ok, I've found a way to get access again:
sudo mv config/packages/local.yaml config/packages/local.yaml.old
sudo bin/console kimai:reload --env=prod

but I confirm that if I give the system-admin permission to a SAML user that permission are lost after de-activation of the login form option.
Is it a bug or design?

**Reply by kevinpapst** [reply]
This is a wrong configuration on your end. You have likely "reset roles on login" activated.

**Comment by amaruzzo** [comment]
Hi Kevin,

you are right!

config/packages/local.yaml
_roles:
resetOnLogin: **true**_

to 

_resetOnLogin: **false**_

Thanks!
